### PR TITLE
Update: Improve error reporting when extending missing config (fixes …

### PR DESCRIPTION
**Pull request by alberto**
…#6115)

**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[X] Add something to the core
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
Improve error message when a base config is not found.

**Is there anything you'd like reviewers to focus on?**
No.

### CODE REVIEW

File: messages/extend-config-missing.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ESLint couldn't find the config "<%- configName %>" to extend from. Please, check the name of the config is correct.
```
Comment (kaicataldo) [inline messages/extend-config-missing.txt]: Nitpick about the wording here: `Please, check the name of the config is correct.` -> `Please check that the name of the config is correct.`

File: messages/extend-config-missing.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.
```
Comment (kaicataldo) [inline messages/extend-config-missing.txt]: Nitpick: Not a big fan of `"If you still can't figure out the problem"` - sounds a bit accusatory. Alternative suggestion: `"If you are still having difficulties after checking the above, please stop by https://gitter.im/eslint/eslint to chat with the team."`

Comment (not-an-aardvark) [inline messages/extend-config-missing.txt]: I know we have other error messages that also link to the chatroom, but have we considered using something other than a hardcoded URL? If we ever switch from Gitter to something else, anyone with an old ESLint version will get directed to an inactive support link. I wonder if we should link people to `eslint.org/chatroom` or something, and then have that URL redirect to Gitter.

Comment (alberto) [inline messages/extend-config-missing.txt]: I can do this as a different PR to avoid mixing concerns.

**Pull request merged**